signup-service/src/test/java/io/mosip/signup/services/RegistrationServiceTest.java

```diff
@@ -643,7 +642,7 @@ public void doVerifyChallenge_toResetPassword_thenKBAChallengeNotFound() {
     }
 
     @Test
-    public void doVerifyChallenge_toResetPassword_thenFetchIdentityFailed() {
+    public void doVerifyChallengeInResetPassword_whenFetchIdentityFail_throwException() {
```

Comment:
seems like duplicate test case, refer line 221, if its not duplicate pls change the name to reflect the testcase